build/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -eu
 
+BIN="$1"
```

Comment:
`$1` ? Or `${GOPATH}/bin/`?

Reply:
I am passing it as a parameter from the makefile, but I need to think about how to make it work from the Dockerfile as well. I asked for a review too early, sorry.